healthcheck/checks.py

```diff
@@ -101,8 +101,12 @@ def items(self):
         return connections.all()
 
     def check_item(self, connection):
-        connection.ensure_connection()
-        db_ok = connection.is_usable()
+        from django.db.utils import OperationalError
+        try:
+            connection.ensure_connection()
+            db_ok = connection.is_usable()
```

Comment:
I would move it out of `try:` block. Looks like  `connection.is_usable()` will never raise such kind of error.

Reply:
moved